docs/metrics/README.md

```diff
@@ -0,0 +1,61 @@
+# OpenTelemetry .NET Metrics
+
+## Best Practices
+
+- When emitting metrics with dimensions, ALWAYS provide the dimensions in an
+  ascending sorted order (order by the dimension key) for best performance.
+
+```csharp
+// Sorted in an ascending order by dimension key: `color` comes before `name`
```

Comment:
should we stick to the term attribute or tag, instead of dimensions?

Reply:
Okay. Let's only use tags.